tshirts/TShirtTests.cpp

```diff
@@ -0,0 +1,15 @@
+#include <assert.h>
+#include <iostream>
+
+#include "TShirtsTests.h"
+#include "TShirtSize.h"
+
+void TShirtsTests::testTShirtSizes() {
+    int minShoulderLengthToTest = 35;
+    int maxShoulderLengthToTest = 45;
+
+    for (int i = minShoulderLengthToTest; i <= maxShoulderLengthToTest; i++) {
```

Comment:
This is a good test... don't give 'invalid' for valid shoulder sizes.
But think of it from a user's perspective as well. Is it enough if they get a t-shirt of any valid size? It will need to fit them as well, no?